src/entrypoint.py

```diff
@@ -72,6 +74,7 @@
 MAX_CTU_DEPTH = "INPUT_MAX_CTU_DEPTH"
 OUTPUT_FILE = "INPUT_OUTPUT_FILE"
 PLATFORM_TYPE = "INPUT_PLATFORM"
+OTHER_OPTIONS = "OTHER_OPTIONS"
```

Comment:
OTHER_OPTIONS = "INPUT_OTHER_OPTIONS"

Reply:
Done (and amended per below). Thanks for spotting and adding the documentation / configuration via the YAML file.